bin/cron/update_hoc_map

```diff
@@ -35,11 +35,9 @@ def main
     # update our JS code (hoc_events_map.js) that visualizes these events
     # to handle missing data.
     # ***
-    # In order to be placed on the map, all events require an organization name,
-    # processed location, and city. With the exception that approved special events
-    # don't need a city so we can show events like the Cocos Keeling Islands.
+    # In order to be placed on the map, all events require an organization name and
+    # a processed location.
     next if data['organization_name_s'].nil? || processed_data['location_p'].nil?
-    next if form[:review] != "approved" && processed_data['location_city_s'].nil?
```

Comment:
Do we still want to skip it if the event isn't approved? (I don't know how approval works...)

Reply:
Closing this look: I learned that `approved` is what indicates that this is a special event. So, this does what the PR description indicates -- removes the requirement for a city from non-special events